.gitignore

```diff
@@ -12,12 +12,6 @@ config.guess
 depcomp
 build
 config.log
-src/gpu/*.mrc
```

Comment:
Are you doing this so that some test images are under version control?

Reply:
Nope, just a leftover as in the comment above. For now, just using the existing embedded images from console test for any testing.